src/nbsphinx.py

```diff
@@ -1796,6 +1797,16 @@ def apply(self):
             env.nbsphinx_files.setdefault(env.docname, []).append(file)
 
 
+class ForceEquations(docutils.transforms.Transform):
+    """Unconditionally enable equations on notebooks."""
+
+    default_priority = 900  # after checking for equations in MathDomain
+
+    def apply(self):
+        env = self.document.settings.env
+        env.get_domain('math').data['has_equations'][env.docname] = True
```

Comment:
being able to set this in `conf.py`, i.e. `nbsphinx_force_mathjax = False` and then

```suggestion
        if nbsphinx_force_mathjax:
            env.get_domain('math').data['has_equations'][env.docname] = True
```

Reply:
Thanks for the suggestion!

I've implemented it in 001db3d52070566a5f9344b7bf6d5f9ad51d17be, but since strictly speaking this isn't limited to MathJax (which is just an implementation detail), I've chosen the more generic name `nbsphinx_assume_equations`.